src/main/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkConfig.java

```diff
@@ -39,6 +39,7 @@ public class ClickHouseSinkConfig {
     public static final String PROXY_PORT = "proxyPort";
     public static final String ZK_PATH = "zkPath";
     public static final String ZK_DATABASE = "zkDatabase";
+    public static final String MAX_RETRIES = "maxRetries";
```

Comment:
Why does the Sink use `retryCount` config instead of the standard `max.retries`? 
see [elasticsearch connector](https://docs.confluent.io/kafka-connectors/elasticsearch/current/overview.html#automatic-retries), [JDBC connector](https://docs.confluent.io/kafka-connectors/jdbc/current/sink-connector/sink_config_options.html#retries)